pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.4.1</version>
+		<version>2.4.3</version>
```

Comment:
You can use `2.4.4` or `2.5.0` (it can cause some problems because this is major release)

Reply:
Fixed